scripts/ci/run-jenkins.sh

```diff
@@ -139,12 +139,34 @@ if [[ ${CI_TAGS} == *'product-sdks-android'* ]];
         echo "IGNORE_PROVISION_MXE=1" >> sdks/Make.config
         echo "IGNORE_PROVISION_LLVM=1" >> sdks/Make.config
         echo "DISABLE_CCACHE=1" >> sdks/Make.config
+        echo "DISABLE_IOS=1" >> sdks/Make.config
+        echo "DISABLE_WASM=1" >> sdks/Make.config
         ${TESTCMD} --label=provision-android --timeout=120m --fatal make -j ${CI_CPU_COUNT} -C sdks/builds provision-android
         if [[ ${CI_TAGS} == *'provision-mxe'* ]]; then
             ${TESTCMD} --label=provision-mxe --timeout=240m --fatal make -j ${CI_CPU_COUNT} -C sdks/builds provision-mxe
         fi
-        ${TESTCMD} --label=llvm --timeout=240m --fatal make -j ${CI_CPU_COUNT} -C sdks/builds provision-llvm-llvm{,win}{32,64}
+        ${TESTCMD} --label=provision-llvm --timeout=240m --fatal make -j ${CI_CPU_COUNT} -C sdks/builds provision-llvm-llvm{,win}{32,64}
         ${TESTCMD} --label=runtimes --timeout=120m --fatal make -j ${CI_CPU_COUNT} -C sdks/builds package-android-{armeabi-v7a,arm64-v8a,x86,x86_64} package-android-host-{Darwin,mxe-Win64} package-android-cross-{arm,arm64,x86,x86_64}{,-win}
+        ${TESTCMD} --label=package --timeout=60m --fatal make -C sdks/android package
+        ${TESTCMD} --label=mini --timeout=60m make -C sdks/android check-mini
```

Comment:
on what arch is it running? armeabi-v7a, arm64-v8a, x86 or x86_64? I guess it's whatever is available via adb?

what if we want to test 32bit and 64bit? it's possible to run an android app targeting armeabi-v7a on an arm64 device.

Reply:
It's right now running on any architecture the emulator is. We should add it later, with the AOT configuration, AOT+LLVM configuration, etc.